lib/pharos/config.rb

```diff
@@ -32,11 +33,14 @@ def self.load(raw_data)
 
       # inject api_endpoint to each host object
       config.hosts.each { |h| h.api_endpoint = config.api&.endpoint }
+      # inject bastion to each host object
+      config.hosts.each { |h| h.configure_bastion(config.bastion) } if config.bastion
```

Comment:
could be just `h.bastion = config.bastion` inside the hosts enumeration above?

Reply:
Now with one loop.